airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java

```diff
@@ -604,11 +604,16 @@ private String wrapAndQuote(final String namespace, final String tableName) {
   @Override
   public String migrateFromV1toV2(final StreamId streamId, final String namespace, final String tableName) {
     return new StringSubstitutor(Map.of(
+        "raw_namespace", StringUtils.wrap(streamId.rawNamespace(), QUOTE),
+        "dataset_location", datasetLocation,
         "v2_raw_table", streamId.rawTableId(QUOTE),
         "v1_raw_table", wrapAndQuote(namespace, tableName)
     )
     ).replace(
-        """      
+        """
+            CREATE SCHEMA IF NOT EXISTS ${raw_namespace}
+            OPTIONS(location="${dataset_location}");
+                 
             CREATE OR REPLACE TABLE ${v2_raw_table} (
```

Comment:
Does this table create statement also need `OPTIONS(location="${dataset_location}")`, or will being in the dataset take care of it?

Reply:
fairly confident that bigquery doesn't let you create tables in a different location than the dataset